src/components/progress/progress.stories.tsx

```diff
@@ -19,17 +22,31 @@ type Story = StoryObj<typeof Progress>;
 /**
  * Default usage example of the Progress component.
  */
-export const Default: Story = {
+export const MD: Story = {
+    args: {
+        value: 10,
+        variant: 'md',
+    },
+};
+
+/**
+ * Thin usage example of the Progress component.
+ */
+export const SM: Story = {
```

Comment:
We don't currently write stories for each variant/size of the component. I would remove these for now and later discuss if/how we want to introduce them (with maybe one story called "Sizes" which renders multiple progress components at once? This would be helpful for components such "Button" which have tons of variants)